src/compiler/scala/tools/nsc/typechecker/Contexts.scala

```diff
@@ -1919,7 +1934,7 @@ trait Contexts { self: Analyzer =>
     def sameDepth: Boolean = imp1.depth == imp2.depth
 
     private def imp2Exists = imp2Ctx.importOrNull != null
-    private def imp1Explicit = imp1 isExplicitImport name
+    def imp1Explicit = imp1 isExplicitImport name
```

Comment:
The usage at L1535 might be extraneous, but keeping this change to be conservative.